### PR TITLE
Add workflows to generate distribution

### PR DESCRIPTION
**Pull request by Purushothaman22**
use github action gps/ncc-gha to generate distribution

### CODE REVIEW

File: .github/workflows/Release.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Build Package to use ncc
+on:
+  push:
+    branches:
+      - master
+jobs:
```
Comment (gps) [inline .github/workflows/Release.yml]: Is this really needed? We will anyway have this file as part of the PR, right?

**Review by gps — CHANGES_REQUESTED** [review]
@Purushothaman22 we still need `dist` to be checked in right? This PR removes it.

File: .github/workflows/Release.yml (hunk 1 above)
Comment (Purushothaman22) [reply .github/workflows/Release.yml]: > @Purushothaman22 we still need `dist` to be checked in right? This PR removes it.

@gps  I removed it intentionally to check if it is working. 

> Is this really needed? We will anyway have this file as part of the PR, right?

Not needed will remove it.

**Review by gps — COMMENTED** [review]
Why this change?  dist/licenses.txt → dist/lincense.txt

**Review by gps — APPROVED** [review]

**Pull request merged**